src/components/existing-request/existing-request-display.vue

```diff
@@ -31,14 +31,20 @@
             <!-- labels and values -->
             <v-col cols="9" class="py-0">
               <v-row dense>
-                <v-col cols="12">
+                <v-col cols="12" class="submitted-date">
```

Comment:
I consider it a best practise to add classnames (or ids) to various blocks on the page (for testing purposes).